src/cmd/prepend.go

```diff
@@ -20,6 +20,19 @@ type prependConfig struct {
 var prependCommand = &cobra.Command{
 	Use:   "prepend <branch>",
 	Short: "Creates a new feature branch as the parent of the current branch",
+	Long: `Creates a new feature branch as the parent of the current branch
+
+Syncs the parent branch (prompts if unknown),
```

Comment:
After trying to add them everywhere I noticed there are tons of edge cases. I'm also not certain describing each and every time GT asks the user for something is helpful documentation. So I removed `(prompts if unknown)` everywhere.